string entity
string value
#float64 confidence_entity